addons/packages/ako-operator/1.4.0/README.md

```diff
@@ -1,14 +1,101 @@
 # ako-operator
 
-This package provides NSX Advanced Load Balancer using ako-operator.
+This package reconciles Cluster API objects and manages the lifecycles of [AVI Kubernetes Operator (AKO)](https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tree/release-1.5.2) using [ako-operator](https://github.com/vmware-samples/load-balancer-operator-for-kubernetes).
 
 ## Components
+* [Deployment](./bundle/config/upstream/akooperator/deployment.yaml) for the Load Balancer Operator (ako-operator)
+* [CRD](./bundle/config/upstream/akooperator/akodeploymentconfig.yaml) for `AKODeploymentConfig`
+* [Two `AKODeploymentConfig` samples](./bundle/config/upstream/akooperator/akodeploymentconfig.yaml) for management and workload clusters respectively
+
+## Supported Providers
+
+The following tables shows the providers this package can work with. Other cloud provider support will be added  
+in the future.
+
+| AWS  |  Azure  | vSphere  | Docker |
+|:---:|:---:|:---:|:---:|
+|  ❌ |   ❌ | ✅  |  ❌  |
+
+## Mailing lists
+* Use [tkg-infrax-akita](mailto:tkg-infrax-akita@groups.vmware.com) to report security concerns to the AKO Team,
+  who uses the list to privately discuss security issues and fixes prior to disclosure.
+* Join the[AKO Distributors](mailto:tkg-infrax-akita@groups.vmware.com) mailing list for early private information and vulnerability disclosure.
```

Comment:
Please remove this, this is an internal mailing list.

Reply:
We might want to create a google group for this instead of using our internal mailing list.